src/Umbraco.Core/Services/SectionService.cs

```diff
@@ -212,7 +212,7 @@ public Section GetByAlias(string appAlias)
         public void MakeNew(string name, string alias, string icon)
         {
             var sections = GetSections();
-            var nextSortOrder = sections != null ? sections.Max(x => x.SortOrder) + 1 : 1;
+            var nextSortOrder = sections != null && sections.Any() ? sections.Max(x => x.SortOrder) + 1 : 1;

```

Comment:
Sorry should have been more clear. If `GetSections()` always returns an enumerable then `Any()`  would be enough. No need for the prior null check. I'd boy scout it if so to make the code easier for the next visitor.